platform/golden/cfg/preseed-debian-11.cfg

```diff
@@ -47,7 +47,7 @@ tasksel tasksel/skip-tasks  string standard
 tasksel tasksel/first multiselect openssh-server
 popularity-contest popularity-contest/participate boolean false
 
-d-i pkgsel/include string curl wget openssh-server neovim sudo
+d-i pkgsel/include string curl wget openssh-server neovim sudo cloud-init
```

Comment:
I think of the golden as the root of an inverted tree. Then, derivative images can be created from that, for specific use cases. All that said, I think the pkgsel should be as minimal as possible.